src/UnigramWordPredictor.java

```diff
@@ -51,7 +53,18 @@ public UnigramWordPredictor(Tokenizer tokenizer) {
   public void train(Scanner scanner) {
     List<String> trainingWords = tokenizer.tokenize(scanner);
 
-    // TODO: Convert the trainingWords into neighborMap here
+    Map<String, List<String>> neighborMap = new HashMap<>();
```

Comment:
This works, but consider just using the instance variable directly instead of creating a new local variable and assigning it at the end. You could just leave off the type declaration to make this work:

```
neighborMap = new HashMap<>();
```